src/ui/components/search/filtersearchcomponent.js

```diff
@@ -138,6 +138,9 @@ export default class FilterSearchComponent extends Component {
   }
 
   onMount () {
+    if (this.autoCompleteComponent) {
+      this.autoCompleteComponent.remove();
+    }
```

Comment:
Kind of! It was more to support removing an applied filter tag created by FilterSearch. Without this line, even if this.setState() is called in FilterSearch, the autocomplete text will not update. This same line was also added to the GeolocationFilter a year ago by Julian, since the components act very similarly I think it should be a safe addition.

Reply:
see https://github.com/yext/answers/pull/825/files/68f2ae3936ef71fac0879aaca7fbe6d615e501a0#diff-8cc525d9efeb6ad87e6063ca5bd7c7a2L171